src/main/java/medicaldoctor/SecondLoginController.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package medicaldoctor;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+import javafx.fxml.Initializable;
+
+/**
+ *
+ * @author Salmanhussain
+ */
+public class SecondLoginController implements Initializable, ParentInterface{
+    ControllerManager cm;
```

Comment:
should this be a private field? also would it be more appropriate to have this ControllerManager stuff in an abstract parent class instead of an interface where  you  have to repeat this field?